config.md

```diff
@@ -85,7 +85,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is allocated for and attached to that process. Defaults to false.
```

Comment:
@wking I think that both approaches are valid and we shouldn't add language to support one but not the other.